docs/diagrams/alias/AliasMappingClassDiagram.puml

```diff
@@ -0,0 +1,36 @@
+@startuml
+!include ../style.puml
+
+skinparam arrowThickness 1.1
+skinparam arrowColor BLACK
+skinparam classBackgroundColor MODEL_COLOR
+skinparam classBorderColor WHITE
+skinparam classFontColor WHITE
+skinparam classAttributeFontColor WHITE
+
+
+class AliasMapping {
+    getAlias(String) : Alias
+    addAlias(Alias) : void
+    deleteAlias(String) : void
+    containsAlias(String) : boolean
+    isReservedKeyword(String) : boolean
+    size() : int
+}
+
+show AliasMapping members
+
+class Alias {
+    aliasName : String
+    command : String
+    {static} isValidName(String) : boolean
+    {static} isValidCommand(String) : boolean
+    getAliasName() : String
+    getCommand() : String
+}
+show Alias members
+
+AddressBook *-right-> AliasMapping
+AliasMapping o-right-> "*" Alias
```

Comment:
Similarly, would suggest changing this to composition.

In addition, perhaps it should be a dotted arrow as the main mode of interaction is through the hashmap, and so, similar to the storage diagrams, it should be a dotted arrow.

Ref: https://nus-cs2103-ay2021s2.github.io/website/se-book-adapted/chapters/uml.html#what-4